tests/e2e/runner/runner.go

```diff
@@ -97,20 +88,34 @@ done:
 
 		outf("{{magenta}}checking custom VM status{{/}}\n")
 		cctx, ccancel := context.WithTimeout(context.Background(), 2*time.Minute)
-		resp, err := cli.Status(cctx)
+		resp, err := cli.Health(cctx)
 		ccancel()
 		if err != nil {
 			cancel()
 			return "", "", err
 		}
 
+		if resp.ClusterInfo == nil {
+			continue
+		}
```

Comment:
Is there a case where `Health(ctx)` returns a nil error and `ClusterInfo` is also nil?

Reply:
currently not. can fix that